### PR TITLE
Add GDevelop as Engine

### PR DESCRIPTION
**Pull request by 4ian**
### SteamDB app page links to a few games using this

https://steamdb.info/app/1307760/
https://steamdb.info/app/1622620/
https://steamdb.info/app/842170/
https://steamdb.info/app/1460910/
https://steamdb.info/app/1342220/

### Brief explanation of the change

Hi 👋
This adds rules for detecting GDevelop (both for GDevelop 5 and GDevelop 4).

### CODE REVIEW

File: rules.ini [hunk 1]
```diff
@@ -53,6 +53,8 @@ CryEngine[] = (?:^|/)engine\.pak$
 CryEngine[] = \.cry$
 Defold = (?:^|/)game\.dmanifest$
 FNA = (?:^|/)fna\.dll$
+GDevelop[] = (?:^|/)gd\.js$
```
Comment (xPaw) [inline rules.ini:56]: This detects `/node_modules/moment/locale/gd.js`

**Comment by xPaw** [comment]
And some of the games you linked don't have any `gd` files in it, so that sucks. Any ideas?

**Comment by 4ian** [comment]
Can you list the games that don't have these files?
GDevelop 4 games will have (lib)GDCpp.so/dll/dylib.
GDevelop 5 games will have gd.js - but I'll try to find another file (I guess we can't have a way to require the presence of multiple files/multiple regexes to pass?).

**Comment by xPaw** [comment]
First two links.

**Comment by xPaw** [comment]
There are no depots that have a `spriteruntimeobject-pixi-renderer.js` file in them. If they're all package, unfortunately we can't access the files inside the package.

**Comment by 4ian** [comment]
Thanks for checking, I was assuming Electron would not package files by default - my mistake. Seems like there is no obvious way then to check for the engine. I'll see if we can make this obvious with a specific file in a future version then.

**Pull request closed without merging**